src/apps/project-request/ProjectRequest.tsx

```diff
@@ -66,7 +66,6 @@ export const ProjectRequest = (props: ProjectRequestProps) => {
           onError={() => {}}
           me={props.user}
           invitations={[]}
-          projects={[]}
```

Comment:
Note: I saw this highlighted in red when I opened `ProjectRequest`, because the `TopBar` component no longer has a `projects` prop.